README.md

```diff
@@ -1,43 +1,24 @@
-# How to set up Grafana on Clever Cloud
+# How to set up Grafana on Clever-Cloud
 
-1. create a new NodeJs application linked to a fork of this repo
-2. in your application environment variables add ```GF_SERVER_HTTP_PORT=8080```, ```GOPATH=/home/bas/<app_id>/go_home``` where <app_id> is the id of your application on clever cloud (found in the top right corner of the console) and ```GRAFANA_VERSION=v7.3.5``` (or any other tag in the official Grafana repo)
-3. you can now build and start the application, be aware however that if you restart it your data will not be persisted (see next section to learn how to persist data)
+1. Use a fork of this repository as a Node.js application. It will download and run Grafana using the root `package.json` file. The install script `build.sh` will download grafana and its plugins and the start one `run.sh` will start the grafana-server binary.
```

Comment:
I think the first item should be `fork this repository`. Then second should be create the application, with a link to https://www.clever-cloud.com/doc/getting-started/by-language/node/#create-an-application-on-clever-cloud